build.Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.17
```

Comment:
It looks good. Thanks! Can we rename `build.Dockerfile` to `Dockerfile.protoc`? I want to put this protoc Dockerfile next to an original Dockerfile when listing files.

Before
```
-rw-r--r--  1 teppei     2629  2 10 09:33 CONTRIBUTING.md
-rw-r--r--  1 teppei      143  1 16 21:17 Dockerfile
-rw-r--r--  1 teppei    11357  8  1  2021 LICENSE
-rw-r--r--  1 teppei     2094  2 10 09:43 Makefile
-rw-r--r--  1 teppei      137  8  1  2021 NOTICE
-rw-r--r--  1 teppei     9780  2 10 09:33 README.md
-rw-r--r--  1 teppei      570  2 10 09:33 build.Dockerfile
```

After
```
-rw-r--r--  1 teppei     2629  2 10 09:33 CONTRIBUTING.md
-rw-r--r--  1 teppei      143  1 16 21:17 Dockerfile
-rw-r--r--  1 teppei      570  2 10 09:33 Dockerfile.protoc
-rw-r--r--  1 teppei    11357  8  1  2021 LICENSE
```